<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QPushButton" name="serialButton">
     <property name="text">
      <string>打开串口</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>波特率</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="BaudRateBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentText">
        <string>2400</string>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>1200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4800</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>38400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>57600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>数据位</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="DataBitsBox">
       <property name="currentText">
        <string>8</string>
       </property>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>校验位</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="verifyBox">
       <property name="currentText">
        <string>No</string>
       </property>
       <item>
        <property name="text">
         <string>No</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Even</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Odd</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Space</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mark</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>停止位</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="StopBitsBox">
       <property name="currentText">
        <string>1</string>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>表地址：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>流量打点</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>681</width>
         <height>30</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="startButton">
          <property name="styleSheet">
           <string notr="true">QPushButton#queenMenuButton:pressed{
background-color: rgba(255, 255, 255, 150);
border:2px solid white;border-radius:10px;
padding-left:6px;
padding-top:6px;
}</string>
          </property>
          <property name="text">
           <string>开始</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopButton">
          <property name="styleSheet">
           <string notr="true">QPushButton#queenMenuButton:pressed{
background-color: rgba(255, 255, 255, 150);
border:2px solid white;border-radius:10px;
padding-left:6px;
padding-top:6px;
}</string>
          </property>
          <property name="text">
           <string>结束</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>681</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>清空</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>681</width>
         <height>25</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QDateEdit" name="dateEdit">
            <property name="dateTime">
             <datetime>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
              <year>2022</year>
              <month>1</month>
              <day>1</day>
             </datetime>
            </property>
            <property name="date">
             <date>
              <year>2022</year>
              <month>1</month>
              <day>1</day>
             </date>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>起始日子</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_2"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>间隔时间(min)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QSpinBox" name="spinBox"/>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>天数</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QSpinBox" name="spinBox_3"/>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>初始流量</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>695</width>
         <height>23</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QTimeEdit" name="timeEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>密集起始时间</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_4"/>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>密集时长(小时)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_5"/>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>流量增量</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_6"/>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>密集采样间隔时间（分钟）</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>681</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>681</width>
         <height>341</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>2014协议</string>
      </attribute>
      <widget class="QPushButton" name="query_2014_Button">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>10</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>查询指令</string>
       </property>
      </widget>
      <widget class="QPushButton" name="add_2014_Button">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>40</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>添加指令</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="query_2014_lineEdit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>611</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="add_2014_lineEdit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>611</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="send_2014_pushButton">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>280</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>发送指令</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="send_2014_lineEdit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>280</y>
         <width>611</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QListView" name="listView">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>320</y>
         <width>641</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="read_2014_pushButton">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>130</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>读取指令</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>300</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>指令解析</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="read_2014_textEdit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>70</y>
         <width>611</width>
         <height>201</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string> 蓝牙广播间隔</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>234</width>
         <height>30</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLineEdit" name="setTimelineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>ms</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="setTimepushButton">
          <property name="text">
           <string>设置时间</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>100</y>
         <width>234</width>
         <height>30</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLineEdit" name="readTimelineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>ms</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="readTimepushButton">
          <property name="text">
           <string>读取时间</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="format">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="myCombox" name="serialBox"/>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_8"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>myCombox</class>
   <extends>QComboBox</extends>
   <header>mycombox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
